packages/odyssey-storybook/src/components/odyssey-mui/Select/Select.stories.tsx

```diff
@@ -236,42 +237,7 @@ const Template: StoryObj<SelectProps> = {
         isDisabled={args.isDisabled}
         isMultiSelect={args.isMultiSelect}
         isOptional={args.isOptional}
-        options={optionsArray}
-      />
-    );
-  },
-};
-
-const ObjectTemplate: StoryObj<SelectProps> = {
-  render: function C(args) {
-    return (
-      <Select
-        label={args.label}
-        hint={args.hint}
-        errorMessage={args.errorMessage}
-        isDisabled={args.isDisabled}
-        isMultiSelect={args.isMultiSelect}
-        isOptional={args.isOptional}
-        options={optionsObject}
-      />
-    );
-  },
-  args: {
-    hint: "Select your destination in the Sol system.",
-  },
-};
-
-const GroupTemplate: StoryObj<SelectProps> = {
-  render: function C(args) {
-    return (
-      <Select
-        label={args.label}
-        hint={args.hint}
-        errorMessage={args.errorMessage}
-        isDisabled={args.isDisabled}
-        isMultiSelect={args.isMultiSelect}
-        isOptional={args.isOptional}
-        options={optionsGrouped}
+        options={args.options}
```

Comment:
We don't even need this template. CSF 3 format for Storybook does this for us. Just remove it completely, and we're good!